client/src/Root.vue

```diff
@@ -10,7 +10,7 @@ export default {
   },
   computed: {
     shouldShowNavigation() {
-      return this.$route.name !== 'login.index'
+      return this.$route.path.indexOf('/auth') === -1
```

Comment:
Auth have nested routes, if you check only route name all other routes make navbar visible